### PR TITLE
Defined "food desert"

### PR DESCRIPTION
**Pull request by erinborders**
I had to make another request so I could add an end of line character

### CODE REVIEW

File: 11ty/definitions/food-desert.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Food Desert
```
Comment (tatianamac) [inline 11ty/definitions/food-desert.md]:
```suggestion
title: food desert
```

File: 11ty/definitions/food-desert.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+slug: food-desert
+defined: true
+speech: noun
+excerpt: an area that has limited access to affordable and nutritious food
+---
+
+A geographic area where residents’ access to affordable, healthy food options (especially fresh fruits and vegetables) is restricted or nonexistent due to the absence of grocery stores within convenient traveling distance.
```
Comment (tatianamac) [inline 11ty/definitions/food-desert.md]:
```suggestion
a geographic area where residents’ access to affordable, healthy food options (especially fresh fruits and vegetables) is restricted or non-existent due to the absence of grocery stores within convenient traveling distance.
```
Would including "farmers' markets" here make sense?

File: 11ty/definitions/food-desert.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+title: Food Desert
+slug: food-desert
+defined: true
+speech: noun
+excerpt: an area that has limited access to affordable and nutritious food
```
Comment (tatianamac) [inline 11ty/definitions/food-desert.md:6]:
```suggestion
excerpt: an area that has limited access to affordable and nutritious food
reading:
  - text: "Food Empowerment Project: Food Deserts"
    href: https://foodispower.org/access-health/food-deserts/
```

File: 11ty/definitions/food-desert.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+---
+
+A geographic area where residents’ access to affordable, healthy food options (especially fresh fruits and vegetables) is restricted or nonexistent due to the absence of grocery stores within convenient traveling distance.
+
+The other defining characteristic of food deserts is socio-economic: that is, they are most commonly found in black and brown communities and low-income areas (where many people don’t have cars). Studies have found that wealthy districts have three times as many supermarkets as poor ones do, [2] that white neighborhoods contain an average of four times as many supermarkets as predominantly black ones do, and that grocery stores in African-American communities are usually smaller with less selection.
```
Comment (tatianamac) [inline 11ty/definitions/food-desert.md]:
```suggestion
According to [Food Empowerment](https://foodispower.org/access-health/food-deserts/), "the other defining characteristic of food deserts is socio-economic: that is, they are most commonly found in black and brown communities and low-income areas (where many people don’t have cars). Studies have found that wealthy districts have three times as many supermarkets as poor ones do, that white neighborhoods contain an average of four times as many supermarkets as predominantly black ones do, and that grocery stores in African-American communities are usually smaller with less selection."
```

File: 11ty/definitions/food-desert.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+Find more information at the [Food Empowerment Project](https://foodispower.org/access-health/food-deserts/)
```
Comment (tatianamac) [inline 11ty/definitions/food-desert.md]:
```suggestion

```
Added this link in additional reading above.

**Review by tatianamac — APPROVED** [review]

**Comment by tatianamac** [comment]
Thank you so much for submitting this definition, @erinborders ! I left you some syntax stuff and a few comments. Let me know if you have any thoughts. 🤗

**Comment by erinborders** [comment]
Thanks for the suggestions - I went ahead and added them!

**Review by tatianamac — APPROVED** [review]
🚀

**Comment by tatianamac** [comment]
Thank you so much for your time and energy on this term! I merged it in. I hope you'll contribute again soon!

**Pull request merged**